src/data-management/fake-data/fake-data-service.ts

```diff
@@ -3,20 +3,21 @@ import { Note, StorageService } from '../types';
 export class FakeDataService implements StorageService {
     private notes: Note[];
 
-    constructor() {
+    constructor(notes: Partial<Note>[]) {
         this.notes = [];
         const date = new Date();
```

Comment:
The date also should be randomized somehow, because grouping notes by date is one of features we need to implement.
But i can think about it in a scope of my `Notes` page/component

Reply:
or to create helper function to generate bunch of fake initial data. Yeap, lets do it in the scope of tasks.